scripts/route_check.py

```diff
@@ -93,103 +100,138 @@ def do_diff(t1, t2):
 def get_routes():
     db = ConfigDBConnector()
     db.db_connect('APPL_DB')
-    print_message(MODE_DEBUG, "APPL DB connected for routes")
+    print_message(syslog.LOG_DEBUG, "APPL DB connected for routes")
     keys = db.get_keys('ROUTE_TABLE')
```

Comment:
This covers the regular route case. But since we also have VNET routes (VNET_ROUTE_TABLE, VNET_ROUTE_TUNNEL_TABLE), it will show mismatch when run on switches with VNETs. Just thinking how we should handle that, or extend this script to cover it as a later commit? @lguohan

Reply:
As this does not affect 20181130, we will commit this PR and take VNET routes handling in a separate PR.